Makefile

```diff
@@ -44,6 +48,10 @@ bin/conmon: src/conmon.o src/cmsg.o src/ctr_logging.o src/utils.o | bin
 %.o: %.c
 	$(CC) $(CFLAGS) -o $@ -c $<
 
+config: git-vars cmd/conmon-config/conmon-config.go runner/config/config.go runner/config/config_unix.go runner/config/config_windows.go
```

Comment:
I guess we have to put it as dependency to either `all` or `src/conmon.o`, right?

Reply:
I'd rather we not have the hard dependency on go. The tool chain is pretty opinionated. I'd rather keep config.h in sync with config.go (by the gating test) so one can presumably compile conmon with just the c tool chain. WDYT

Reply:
Yeah I also thought that this would be a valuable solution. SGTM